benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -341,10 +354,13 @@ def get_body(vec):
 
         results = {}
         query_responses = []
-        for v in self.dataset.test:
+        while True:
+            query = self.dataset.read(1)
```

Comment:
nit: constant with name why it is 1

Reply:
1 because it reads 1 vector from the dataset at a time.